smoke-test/action.yml

```diff
@@ -39,12 +42,59 @@ inputs:
 runs:
   using: "composite"
   steps:
+  - name: Extract Chart YAML
+    id: extract-chart-yaml
+    if: inputs.version == ''
+    shell: bash
+    run: |
+      tarball=$(find ${YAML_DIR} -name "${CHART_NAME}*.tgz")
```

Comment:
I guess this will break if the `yaml-dir` doesn't contain a helm chart?

Reply:
yes, all the manipulation of the chart will break, I should make those conditional

Reply:
made a few more things conditional